harmony/request.py

```diff
@@ -420,7 +426,15 @@ def __init__(self,
         ]
         self.parameter_validations = [  # for simple, one-off validations
             (True if self.destination_url is None else self.destination_url.startswith('s3://'),
-             ('Destination URL must be an S3 location'))
+             'Destination URL must be an S3 location'),
+            (self.concatenate is None or isinstance(self.concatenate, bool),
+             'concatenate must be either True of False'),
```

Comment:
'True of False' typo in several places.

Since we are requiring a boolean type (`pixel_subset='True'` fails validation), we should say something like:
'concatenate must be a boolean type either True or False'

Reply:
Updated message to `concatenate must be a boolean (True or False)`